.buildkite/pipeline.deploy.yml

```diff
@@ -0,0 +1,49 @@
+---
+env:
+  PANTS_CONFIG_FILES: "['pants.toml', 'pants.ci.toml']"
+  BUILDKITE_PLUGIN_VAULT_ENV_SECRET_PREFIX: "secret/data/buildkite/env"
+
+steps:
+  - label: ":pulumi: Update grapl/customer-mock1 environment in customer-mock1 account"
+    plugins:
+      - grapl-security/vault-login#v0.1.0
+      - grapl-security/vault-env#v0.1.0:
+          secrets:
+            - PULUMI_ACCESS_TOKEN
+      - improbable-eng/metahook#v0.4.1:
+          post-command: |
+            .buildkite/scripts/grapl_testing_stack/dump_artifacts.sh
```

Comment:
I don't think this plugin is necessary here.

Reply:
If it turns out that we need these logs, this is easy enough to add back in. And frankly they're probably available at a different artifact path anyway. Removing

Reply:
These logs are generated from our integration / e2e tests, though, not Pulumi.